base/util.jl

```diff
@@ -301,8 +301,9 @@ end
 
 function with_output_color(f::Function, color::Union{Int, Symbol}, io::IO, args...)
     buf = IOBuffer()
+    buf_context = IOContext(io, buf)
```

Comment:
Updated. Testing `print_with_color` is kinda awkward because it only looks at the global variable in base. I evaled into it but maybe that is not ok.